src/components/ContentAccordion.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <button
+      class="expansion-button"
+      :aria-controls="'expansion-content-' + ariaTitle"
+      :id="'expansion-control-' + ariaTitle"
+      @click="togglePanel"
+    >
+      <div class="w-full">
+        <div class="accordion-title pb-1">
+          <slot name="title">
+            {{ title }}
+          </slot>
+        </div>
+        <div class="accordion-description pb-1" v-if="description">
+          <slot name="description">
+            {{ description }}
+          </slot>
+        </div>
+        <div class="flex items-center">
+          <span class="accordion-description">
+            {{ showPanel ? hideText : showText }}
+          </span>
+          <span class="icon-wrapper">
+            <span
+              class="panel-icon"
+              :class="{
+                'show-panel-icon': showPanel,
+                'hide-panel-icon': !showPanel
+              }"
+            ></span>
+          </span>
+        </div>
+      </div>
+    </button>
+    <transition name="slide-fade">
+      <div
+        class="panel-content"
+        :aria-hidden="!showPanel"
+        :id="'content-' + ariaTitle"
```

Comment:
```suggestion
        :id=`content-${ariaTitle}`
```
Use interpolation rather than concatenation. It's a bit clearer